database/rrd.h

```diff
@@ -764,6 +764,8 @@ struct rrdhost {
     const char *os;                                 // the O/S type of the host
     const char *tags;                               // tags for this host
     const char *timezone;                           // the timezone of the host
+    const char *abbrev_timezone;                    // the abbriviated timezone of the host
+    int32_t utc_offset;                             // the offset in seconds from utc
```

Comment:
`rrdhost` is the main structure of our data model, did you test this PR  on a machine that had an older netdata installed? I have the feeling that an update will break compatibility. :thinking:

Reply:
Hi Thiago.

Yes, I did some tests (streaming) with old (i.e. without this PR) agents. 

If an old agent is connected as child to a new parent  (with this PR), then for the child `rrdhost` we will have `UTC` as an abbreviated timezone, and `0` as `utc_offset`. (i.e. the child does not send these new attributes).

If a new agent is connected as a child to an old parent, then the new attributes that the child sends will be ignored by the parent.

I've documented this behaviour in the Test Plan. Is that what you're asking? Perhaps you mean something else?

Reply:
Hello @MrZammler ,

My concern in this PR and PR #11059 (ping @stelfrag ) is when users start using netdata after these PRs are merged, we load host from our database and instead to get the stored values in `RRDHOST_FLAGS flags;` and ` RRDHOST_FLAGS *exporting_flags;`, they will have values stored in these new variables. 

I am already months without to see what is happening in our dbengine, so it is possible I am missing something and my concern is not necessary.

Reply:
Hi Thiago, yes I see what you mean. Yes, the new `utc_offset` and `abbrev_timezone` are not stored in the sqlite. At this point I think it's not needed (but @stelfrag will of course decide).

There is a change in `database/sqlite/sqlite_functions.c` (line 987), where when a host is read from the sqlite, in `set_host_properties` it will pass `0` for `utc_offset` and `NULL` for `abbrev_timezone` (which will be `UTC` eventually).

I think it is safe for now, and if at a later date we decide to also store these attributes in sqlite, it can be done.

Reply:
The system info is not currently stored in the database so it is not affected